Edit Group Modal Alignment

brianHarder opened the pull request:
<!-- You can remove this first section if you have contributed before -->

### First time contributor checklist:

- [X] I have read the [README](https://github.com/signalapp/Signal-Desktop/blob/main/README.md) and [Contributor Guidelines](https://github.com/signalapp/Signal-Desktop/blob/main/CONTRIBUTING.md)
- [X] I have signed the [Contributor Licence Agreement](https://signal.org/cla/)

### Contributor checklist:

- [X] My contribution is **not** related to translations.
- [X] My commits are in nice logical chunks with [good commit messages](http://chris.beams.io/posts/git-commit/)
- [X] My changes are [rebased](https://medium.com/free-code-camp/git-rebase-and-the-golden-rule-explained-70715eccc372) on the latest [`main`](https://github.com/signalapp/Signal-Desktop/tree/main) branch
- [X] A `pnpm run ready` run passes successfully ([more about tests here](https://github.com/signalapp/Signal-Desktop/blob/main/CONTRIBUTING.md#tests))
- [X] My changes are ready to be shipped to users

### Description

This PR fixes the alignment issue in the edit group modal. I did this by adding the 'expandable' property to GroupTitleInput.tsx. This matches GroupDescriptionInput.tsx, which also has this property. Users now have a properly formatted window for editing groups.

I manually tested this change by loading my Signal data into my staging environment. I then accessed the edit group modal in a group to observe the change. This was done on my laptop, which is running macOS 10.15.7.

Current:
![image](https://github.com/user-attachments/assets/af4b0f07-d335-4fce-8df8-83ef1f396237)

Changed:
![image](https://github.com/user-attachments/assets/9471cabb-7c6d-4b5c-96e4-47aebf0ac7f0)

trevor-signal commented:
Thanks for this PR, @brianHarder! I agree that this should be fixed, and I'm just checking with our design team to make sure that we want to standardize to the smaller amount of padding.

I'll let you know when I hear back.

In any case, I think we'll want to address this in a slightly different manner -- while this PR does fix the problem, I don't love adding an `expandable` prop to a text input that we don't intend to actually be expandable.  In practice it does not expand because of the character limit, but I'd love to be strict with the semantics of the properties we are passing. For example, in the future someone might change something for all `<Input expandable ... />`s in the future and they may not realize it also is applying to this, non-expandable, one.

brianHarder commented:
@trevor-signal Thanks for reaching out to the design team! I removed the expandable prop in the GroupTitleInput and updated Input.scss with 12px padding. Here's what it looks like now:

<img width="1324" alt="EditGroup_Updated" src="https://github.com/user-attachments/assets/a9938048-11f4-4ddf-b638-c619eab6d2cb" />
